### ISSUE TITLE
Issue with the grading criteria

### ISSUE THREAD

**Issue by waynchi** [usc-csci201-fall2013/simcity201#42]
Dear Professor Wilcynski,

After reading your take on the extra credit, I have several concerns that I would like to voice.

1) You say that the person working on A\* or Databases will be able to mentor other team members so that they may implement A\* or Databases as extra credit as well. However, there is absolutely no incentive to doing so (grade wise). Rather than spending his valuable time helping out his team member do extra credit he could instead be furthering his own individual project portion, making this mentoring principle completely reliant on his own good will.

2) If say, I assigned someone the task of A\* or Databases, he will be spending his time that could be spent benefiting everyone purely benefiting himself, which defeats the purpose of a team effort.

3) There are certain parts of the project that purely cannot do some of the extra credit! For example, it's impossible for the person doing the main Gui to do the Databases extra credit, which makes it unfair for him as he loses an extra credit opportunity.

4) Team Leaders are no longer as heavily rewarded for organizing the team. Before, the non-menial task of organizing the entire team, assigning tasks, scheduling meetings, among others was extra work, but rewarding because it would directly affect the Team Leader's entire grade. Now, that reward is cut in half because this effort will have absolutely no impact on the Team Leader's individual grade.

5) Because this team project is supposed to be fluid and not fixed, members should be able to switch between portions of the project easily to fulfill the every changing requirements. However, this is no longer the case as then their previous work will be counted into another person's individual grade. For example, if one person was originally working on markets, but had to be reassigned to Gui, his work on the market would suddenly no longer benefit him, but benefit the other person's individual grade.

6) Since this is a TEAM project, everyone on the group should be working together, trying to help each other out to get the project to be the best it can possibly be. However, with the addition of individual credit, the "helper" will basically be putting his time and effort directly into someone else's individual credit, making it less likely that he will want to help out. 
Basically, instead of helping out those who are harder workers and better coders with the individual grade portion (which is what I assume you were trying to do since you have the 70% rule), you're actually making it harder on them! They still have to help other team members, but they get less for doing so (since it will not affect their individual grade).

Essentially, I feel as though this new emphasis on individual scores does not benefit the team project as a whole, and changes it from a team project setting to a "do your individual code and then put it all together" setting.

Thank you for your time, and have a good weekend! :)

Sincerely,
A Concerned Student

**Comment by ztanner** [comment]
I would like to reiterate the points that waynchi made, especially the last two. This new grading schema incentivizes individual work, when in fact it has been the case that members are helping contribute to and fix bugs with other group member's code.  Based on the new grading breakdowns, it sounds like such members will be penalized for not focusing heavily on their portion of the project, which doesn't make any sense.

**Comment by khupka** [comment]
@dwilczyn, I also agree with @waynchi and @ztanner.

By introducing individual extra credit, if a person were to finish all of his/her duties, instead of helping/mentoring other group members to complete unfinished parts, he/she would focus only on improving his/her own code.  Individual extra credit defeats the purpose of working in groups and in teaching us the larger scheme of the importance of group work.

If extra credit is given, it should be given to the group.  This would also help ensure all other requirements are met and students have an understanding of what is necessary before trying to implement additional features.

**Comment by dwilczyn** [comment]
Good points, all of them. I will discuss them with Prof. Crowley next week.
